src/Raven.Client/Documents/Session/AsyncDocumentQuery.cs

```diff
@@ -635,6 +636,12 @@ public IAsyncDocumentQuery<TProjection> SelectFields<TProjection>(QueryData quer
             return CreateDocumentQueryInternal<TProjection>(queryData);
         }
 
+        public IAsyncDocumentQuery<TTs> SelectTimeSeries<TTs>(Func<ITimeSeriesQueryBuilder, TTs> timeSeriesQuery)
```

Comment:
Let's change to explicit interface implementation. It is way easier to track code changes then when interface changes.

Reply:
done